File: OnlineJudge/P10071.java
import java.util.Scanner;

public class P10071 {

    public static void main(String[] args) {
        
        Scanner s = new Scanner(System.in);

        while (s.hasNextInt()){

            int v = s.nextInt();
            int t = s.nextInt();

            if (t==0){

                System.out.println(0);

            } else {

                double accel = v/(t*1.0);

                double pos = (accel*(t*2)*(t*2))/2.0;

                System.out.println(Math.round(pos));

            }

        }

    }

}

File: OnlineJudge/P10071.c

#include <stdio.h>
#include <math.h>

int main()
{

    int v, t;
    while (scanf("%d %d", &v, &t)==2){

        if (t==0){

            printf("0\n");


        } else {

            double a = v/(t*1.0);
            double pos = (a*(2*t)*(2*t))/2;

            int res = (int)(pos < 0 ? (pos - 0.5) : (pos + 0.5));;

            printf("%d\n", res);

        }

    }

    return 0;

}